packages/trip-details/src/fare-table.tsx

```diff
@@ -142,9 +96,18 @@ const FareTypeTable = ({
             {boldText(getFormattedTextForConfigKey(header))}
           </th>
           {colsToRender.map(col => {
-            const fare = fareTotals[col.key];
+            let fare;
+            if (!hasLegProducts) {
+              fare = fareTotals[col.key];
+            } else {
+              fare = getItineraryCost(
+                legs,
+                col.riderCategory,
+                col.fareContainer
+              );
+            }
```

Comment:
Usually I prefer blocks to a ternary but I feel in this situation a `const` would be cleaner

Reply:
This also makes the typescript work better